metal/roles/cronjobs/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Install Cronie Package
+  dnf:
+    name: cronie
+    state: present
+
+- name: Set up FIRST WEEKLY reboot
+  cron:
+    name: First Weekly Reboot
+    minute: 0
+    hour: 2
+    weekday: 1
+    job: "/sbin/reboot"
+    state: present
+
+- name: Set up SECOND WEEKLY reboot
+  cron:
+    name: Second Weekly Reboot
+    minute: 0
+    hour: 2
+    weekday: 5
+    job: "/sbin/reboot"
+    state: present
```

Comment:
I did a little more chasing here:

By default, the Kured deployment is set to run a sentinel command once an hour which returns an exit code based on the nodes need/desire to reboot, whether that be pending packages/dependencies or an OS update, if either of those events (or anything that sets the flag on the sentinel command) take place, then Kured will schedule a reboot on that node.

What's nice is that the sentinel command is actually not part of kured but is actually a component of the Fedora package manager itself (`dnf`); Kured is simply taking advantage of this feature to know when to reboot a node. You can test this locally on a node or any RHEL based system for that matter by simply running `needs-restarting --reboothint` which will tell you if anything has happened on that system that requires a reboot. If you run it this way, the command will yield an exit code of `1`. Kured flips this on its head:
>by default the command will return 1 if a reboot is required, so we wrap it in sh -c and add ! to negate the return value

The issue is that if you've got Kured set to run the sentinel command, checking for the sentinel file won't work.
> If a reboot sentinel command is used, the reboot sentinel file presence will be ignored.

However, we can do something really clever that will allow us to leverage both features simultaneously. Say for example we copy the below script to the nodes upon provisioning them (Another step/role in the Ansible playbook for `metal` would be ideal):

```sh
#!/bin/bash

# Check if /var/run/reboot-required exists
if [ -f "/var/run/reboot-required" ]; then
    exit 0
fi

# Alternatively, use needs-restarting to check if a reboot is needed
if ! needs-restarting --reboothint; then
    exit 0
fi

# Default exit, meaning no reboot required
exit 1
```

We can then set kured to call the above script instead of `sh -c "! needs-restarting --reboothint"` at a 5 minute interval. This way when the script runs, it will check for any flags that the package manager might have placed requiring a node reboot as well as the presence of a sentinel file. It can be extended further with your cronjob by simply running `sudo touch /var/run/reboot-required` at the interval/time you'd like which will trigger the node cordon, drain and reboot.